redisinsight/desktop/src/lib/cloud/cloud-oauth.handlers.ts

```diff
@@ -37,6 +68,29 @@ export const getTokenCallbackFunction = (webContents: WebContents) => (response:
 export const initCloudOauthHandlers = () => {
   ipcMain.handle(IpcInvokeEvent.cloudOauth, async (event, options: CloudAuthRequestOptions) => {
     try {
+      if (process.env.NODE_ENV === 'development') {
```

Comment:
please change everywhere `process.env.NODE_ENV === 'development'` to `config.devBuild` or `config.isDevelopment`